pkg/operator/staticresourcecontroller/static_resource_controller.go

```diff
@@ -0,0 +1,211 @@
+package staticresourcecontroller
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+
+	corev1 "k8s.io/api/core/v1"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
```

Comment:
nit: order

Reply:
@deads2k @sttts you have to watch https://www.youtube.com/watch?v=kAM-YW-6vdU :D

Reply:
![download-1](https://user-images.githubusercontent.com/730123/72989813-48700380-3def-11ea-96ab-96efc66403a6.jpg)